integration-tests/elytron-resteasy-reactive/src/main/resources/application.properties

```diff
@@ -7,3 +7,4 @@ quarkus.security.users.embedded.users.poul=poul
 quarkus.security.users.embedded.roles.poul=interns
 quarkus.security.users.embedded.plain-text=true
 quarkus.http.auth.basic=true
+employees-config-property=employees
```

Comment:
I'm not sure I see what is happening here, the authenticated user must have an `interns` role, how does `employees-config-property=employees` align with it ?

Reply:
please check this line @sberyozkin https://github.com/quarkusio/quarkus/blob/main/integration-tests/elytron-resteasy-reactive/src/main/resources/application.properties#L3

Reply:
Authenticated user that needs access to `employee` needs `employees` role.

Reply:
Oh, sorry, I forgot that the actual enhancement is about being able to vary the roles allowed value (for a sec I thought that property was kind of disconnected from the actual namespaced authentication configuration, ex, for the basic auth - but it is really about controlling dynamically users in which role can access this method)